### ISSUE TITLE
On iPhone 7/iOS 15, tapping to expand ALT text doesn't work on some photos

### ISSUE THREAD

**Issue by surfdude29** [bluesky-social/social-app#2751]
**Describe the bug**
In 1.67 on iOS, when I tap to expand ALT text it is not all shown.

**To Reproduce**

1. Open a post with an image with long ALT text, e.g.: https://bsky.app/profile/maryscottoconnor.com/post/3kkes4kov6525
2. Tap on the ALT text to expand it.
3. Only part of the ALT text is shown.

**Expected behavior**

All of the ALT text is shown.

**Screenshots**

![042E4CEE-013A-4B73-B9C5-60EE2F63F181](https://github.com/bluesky-social/social-app/assets/149612116/21142e59-5c93-4dbf-8f60-30a4eb3c263e)
![55907B09-EEC6-467E-B218-A8EA319C74E1](https://github.com/bluesky-social/social-app/assets/149612116/d9eb55e2-885f-44fb-bbab-c7e4b48e8f74)

**Details**

 - Platform: iOS
 - Platform version: 15.8.1
 - App version: 1.67

**Comment by surfdude29** [comment]
I did some more investigating using maryscottoconnor's media tab, as she reliably posts long ALT text and her photos are SFW (I first noticed it on NSFW content).

Also, I deleted the app, rebooted my phone and then reinstalled.

And for reference this is an iPhone 7 running iOS 15.8.1 that I'm testing on.

The bug appears on these posts:

https://bsky.app/profile/maryscottoconnor.com/post/3kkes4kov6525

https://bsky.app/profile/maryscottoconnor.com/post/3kkhhzuw4bc2n

https://bsky.app/profile/maryscottoconnor.com/post/3kkgkdncrkc26

https://bsky.app/profile/maryscottoconnor.com/post/3kkeptuytbu2g

https://bsky.app/profile/maryscottoconnor.com/post/3kk26jlomel2s

https://bsky.app/profile/maryscottoconnor.com/post/3kjxj3gk7og2q (second photo)


But *not* on these posts:

https://bsky.app/profile/maryscottoconnor.com/post/3kkgpykyldk2l

https://bsky.app/profile/maryscottoconnor.com/post/3kkdxwurx3v2e

https://bsky.app/profile/maryscottoconnor.com/post/3kk4qucqxgl27

https://bsky.app/profile/maryscottoconnor.com/post/3kk2seamc5d2w

https://bsky.app/profile/maryscottoconnor.com/post/3kjvrkesbcd2n

https://bsky.app/profile/maryscottoconnor.com/post/3kjv3lvlxzt2n

I haven't yet noticed a pattern as to what's similar or different between the two sets.

**Comment by surfdude29** [comment]
One more data point to add: I can view all the ALT text on all the posts linked above fine when using the bsky.app website on my phone, it's just the app that's affected.

**Comment by haileyok** [comment]
Thanks. I'm going to cross my fingers this isn't an iOS <= 15 issue, will look into it soon.

**Comment by surfdude29** [comment]
Confirmed on my iPhone 7 to be fixed by #2956 in 1.69, thanks @haileyok ! 😊
